torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2964,6 +2981,29 @@ def sample_inputs_entr(op_info, device, dtype, requires_grad, **kwargs):
                                     low=low,
                                     requires_grad=requires_grad)))
 
+
+def sample_inputs_i0_i1(op_info, device, dtype, requires_grad, **kwargs):
+
+    samples = (SampleInput(make_tensor((S,), device, dtype,
+                                       requires_grad=requires_grad)),
+               SampleInput(make_tensor((), device, dtype,
+                                       requires_grad=requires_grad)))
+
+    if dtype.is_floating_point:
+        if op_info.op != torch.special.i0e:
+            # Second-order derivative for `i0e` at `0`
```

Comment:
This comment suggests the code applies to i0e, but the if statement applies to everything BUT i0e

Reply:
Have updated the comment, hopefully it is clearer now.